unity/2017.4.40f1/Makefile

```diff
@@ -1,6 +1,7 @@
+UNITY_CHANGESET := 6e14067f8a9a
```

Comment:
Needed to make sure the unity changeset was defined before the `ifdef UNITY_CHANGESET` in `common.mk` was reached.